src/widgets/Container/index.js

```diff
@@ -3,17 +3,34 @@ import Markdown from '../Markdown';
 import styles from './styles.css';
 
 export default class Container extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      jsonView: false
+    }
+
+    this.handleOnClick = this.handleOnClick.bind(this);
+  }
+
+  handleOnClick(event) {
+    if (event) {
```

Comment:
When will `event` not be defined? If it always defined then we can rewrite this method as

```js
handleOnClick = () => {
  this.setState({
    jsonView: !this.state.jsonView
  });
};
```